docs/getting_started_user.md

```diff
@@ -1,19 +1,37 @@
 # Getting started with PROJECTNAME
 
-This guide takes you through a simple example to get a first quick success with PROJECTNAME. 
+Projects should provide users with a getting started guide.
+The purpose of a getting started guide is to increase adoption and remove barriers to entry for your project.
+Users should be able to complete the steps in approximately five minutes.
 
-1. Install PROJECTNAME.
+Your project's getting started guide should give users a first quick success.
+In general, user should install the project, quickly verify the installation, and run one or two commands.
+The commands in your project's getting started guide should do something basic but illustrate your project's core functionality.
+If appropriate, consider provide sample configuration values and artifacts that allow users to run code locally.
 
-```
-pip install PROJECTNAME
-```
+> The following steps are intended to give an idea of a reasonable getting started guide with an example project named "ansible-project".
```

Comment:
I like how the `>` makes the text different on a preview. Do you think it's worth doing that to the whole intro above or overkill?

Reply:
I feel like that would be overkill.